dependencies/pre-deployment-pvc-binding/pre-deployment-pvc-binding.yaml

```diff
@@ -0,0 +1,36 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: test-pvc-ns
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: test-pvc
+  namespace: test-pvc-ns
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
+  storageClassName: standard
```

Comment:
I'm rethinking this, and I think we should drop the storage class from here. I think on Kind, we're using `standard` all over because this is the default storage class, but on other setups this might be different. We should let the storage class to be the cluster's default.